### PR TITLE
adding notes and Links to ep25 of geeksblabla

### PR DESCRIPTION
**Pull request by ylizma**
#23

### CODE REVIEW

File: blablas/ep25/index.md [hunk 1]
```diff
@@ -23,10 +23,67 @@ In this episode of GeeksBlabla, we discuss with amine and geeksblabla team java
...
+
+0:43 - What are the Java licensing changes?
+
+0:47 - What are the best practices for modern industrial Java projects ?
+
+0:50 - What is JVM in general ?
+
+0:56 - What are the most important Java Concepts ?
+
+0:59 - What is the best Java frsmework ?
```
Comment (yjose) [inline blablas/ep25/index.md]: framework

File: blablas/ep25/index.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ url: ""
 podcastUrl: ""
 ---
 
-In this episode of GeeksBlabla, we discuss with amine and geeksblabla team java and a lot of topics around it
+In this episode of GeeksBlabla, Mohammed, Amine, and Amine talk about Java language, and why you should learn it, they talk also about the strength of The JVM, the best practices as Java Devloper.
```
Comment (yjose) [inline blablas/ep25/index.md]: Java developer*

File: blablas/ep25/index.md (hunk 1 above)
Comment (ylizma) [reply blablas/ep25/index.md]: done

File: blablas/ep25/index.md (hunk 2 above)
Comment (ylizma) [reply blablas/ep25/index.md]: done

**Comment by yjose** [comment]
@all-contributors please add @ylizma  for content

**Pull request merged**